client/src/utils/interfaces.tsx

```diff
@@ -4,3 +4,17 @@ export interface Credentials {
   username: string;
   password: string;
 }
+
+export interface Document {
+  id: number | string;
+  title: string;
+  description: string | null;
+  stakeholder: string | undefined;
+  scale: string | undefined;
+  issuanceDate: Date | null;
+  type: string | undefined;
+  connections: string | undefined;
```

Comment:
connection is likely a number

Reply:
Ok but this wouldn't match the API draft, where connections is links = [targetDocumentId: number, connectionType: string]. Let connections a number it's easier for me to manage and to show. The correct parameter for the API will be managed in an other way.

Reply:
export interface Link {
  targetDocumentId: number;
  type: string[];
}

export interface Document {
  id: number | string;
  title: string;
  description: string | null;
  stakeholder: string | undefined;
  scale: string | undefined;
  issuanceDate: Date | null;
  type: string | undefined;
  connections: Link[] | undefined;
  language: string | undefined;
  pages: number | null;
  coordinates: { latitude: number | null; longitude: number | null };
}

Reply:
Most likely this was just a misunderstanding because I called link the complete representation of what happens between two nodes and connections the count of outgoing links from a node